apps/look-and-feel-stories/src/ItemProgressBar/ItemProgressBar.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { ItemProgressBar } from "@axa-fr/design-system-look-and-feel-react";
+import type { Meta, StoryObj } from "@storybook/react";
+import { ComponentProps } from "react";
+
+const meta: Meta<typeof ItemProgressBar> = {
+  component: ItemProgressBar,
+  title: "Components/ItemProgressBar",
+  parameters: {
+    layout: "centered",
+  },
+  argTypes: {
+    value: {
+      control: {
+        type: "number",
+      },
```

Comment:
```suggestion
      control: { type: "range", min: 0, max: 100, step: 10 },
```
Je pense que ce control est plus adapter au progessbar 😉

![Image](https://github.com/user-attachments/assets/799933b6-7d7c-4599-9187-16d1cc8acd15)

Reply:
Au départ je voulais faire cela mais je me suis dit que comme l'utilisateur pouvait modifier le max cela serait gênant de bloquer la value de 0 à 100

Reply:
oui mais si tu suis le zeroheight tu ne peux avoir que les valeur 0, 10, 20, 30, 40, 50, 60, 70, 80, 90, et 100.
du coup je me demande si c'est utile de changer le max

Reply:
Non, sur le zeroheight la valeur est en pourcentage (et donc dit de passer de 10% en 10%), sur la story on parle de la value qui est une valeur brute par rapport au max.
Je pense que si on veut faire cela il faudrait mettre en valeur de max à 100 sur la story.